packages/table/src/Row.tsx

```diff
@@ -70,42 +79,46 @@ const modeStyles = {
 };
 
 const rowStyle = css`
+  border-color: var(--lg-table-row-border-color);
+  border-top-width: 1px;
+  border-top-style: solid;
+
   & > td > ${tdInnerDiv.selector} {
     min-height: 40px;
-    transition: all 150ms ease-in-out;
+    max-height: unset;
   }
 `;
 
 const hideRow = css`
   opacity: 0;
 `;
 
-const transitionStyles = (mode: Mode) => {
-  return {
-    default: css`
-      transition: border 150ms ease-in-out;
-      border-top-color: transparent;
-
-      & > td {
-        padding-top: 0px;
-        padding-bottom: 0px;
-      }
-
-      & > td > ${tdInnerDiv.selector} {
-        max-height: 0;
-      }
-    `,
-
-    entered: css`
-      border-top-color: ${mode === Mode.Light
-        ? uiColors.gray.light2
-        : uiColors.gray.dark1};
+const nestedRowInitialStyle = css`
+  transition: all ${transitionTime}ms ease-in-out;
+  transition-property: border-color, transform, opacity;
+  transform-origin: 50% 0%;
+  border-color: var(--lg-table-row-border-color);
+  opacity: 0;
+  transform: translateY(-10%);
```

Comment:
`translate`, `translateX`, and `translateY` don't utilize the GPU like `translate3d` does. We should most always opt for `translate3d` when performing a transition using a translate transform, even though the declaration is a bit more clunky, since the transition will usually perform at a higher FPS.